### ISSUE TITLE
IcoMoon font icons are not displayed on certain Android devices

### ISSUE THREAD

**Issue by fwilkens** [NativeScript/NativeScript#5044]
### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?

Searched, nothing found.

### Tell us about the problem

IcoMoon icons are not displayed on certain Android devices, specifically noticed missing icons on:

* Samsung Galaxy S8 (7.0)
* Samsung Note 8

The icons _are_ displayed correctly on various other Android devices, for example:

* Google Pixel (7.1.1)
* Moto X (4.4.4)

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 3.3.0
- Cross-platform modules: 3.1.1
- Runtime: tns-android, 3.3.0
- Plugin(s): none

### Please tell us how to recreate the issue in as much detail as possible.
* Clone my repo here https://github.com/fwilkens/NSHelloWorld (it's a hello world template app).
* Build and run the app on a Moto X or Google pixel (both known to work correctly for this issue, at least on an emulator). Notice the font icon is displayed.
* Build and run on a Samsung Galaxy 8. Notice the font icon is missing. I've confirmed the issue for this device emulating via Genymotion. We also have reports of a Samsung Note 8 presenting similar issues (I do not have this device available to confirm that).

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

The code in the repo linked above has everything needed.

**Google Pixel**
![pixel](https://user-images.githubusercontent.com/4120346/32523810-ce52d116-c3d1-11e7-83d8-c6396e7c77cb.png)

**Galaxy S8**
![s8](https://user-images.githubusercontent.com/4120346/32523811-ce7090c0-c3d1-11e7-883d-2f7878090f40.png)

**Comment by tsonevn** [comment]
Hi @fwilkens,
Thank you for contacting us,
I tested the provided sample project on my side, however, was unable to recreate a problem. I tested it on Samsung Galaxy S8 with Android 7and the icon seems to be loaded properly. For your convenience, I am attaching a screenshot of the device. 
I notice that you are using old tns-core-modules, change the version of `tns-core-modules` to 3.3.0 in the project package.json file. 
Delete platforms and node_modules folders and rebuild the application.

![daemonnotrunning startingitnowonport5037 daemonstartedsuccessfully daemonnotrunning startingitnowonport5037 daemonstartedsuccessfully nrd90mntsonev11082017125650](https://user-images.githubusercontent.com/17448734/32545748-62c1d9a6-c485-11e7-9c09-91cc8261badc.png)

**Comment by fwilkens** [comment]
Thanks for the quick reply @tsonevn. I've updated tns-core-modules to 3.3.0, rebuilt the project, and am still seeing the issue on the emulator. It looks like you may be on a real device? I'd be curios to know if you run into the issue on an emulator of a Galaxy s8?

I'll continue looking into the issue. Thanks for your help!

**Comment by fwilkens** [comment]
Hey @tsonevn, I've done some more digging and have an update on how to re-create this issue more reliably. It appears to happen in high DPI/font-size contexts. I've been able to re-create the issue on several devices now. This is an emulated Google Pixel (640 dpi emulated via Genymotion). At this default dpi, the icon does not appear. When the system font size is reduced, the icon appears again.

You may be able to re-create on your Galaxy S8 by cranking up the font size to the maximum.

![ns-icon-issue](https://user-images.githubusercontent.com/4120346/32962018-ad86f6c8-cb7f-11e7-8bc7-8b3c026a2513.gif)

**Comment by NickIliev** [comment]
@fwilkens using Nexus 5x and applying the text size to maximum but still not being able to reproduce the issue. Can you test with a real device instead of GenyMotion and let me know if the problem persist?

Closing due to inactivity but do let me know if the issue is still reproducible with the latest NativeScript.

**Comment by fwilkens** [comment]
Hi @NickIliev,

I should've clarified my last update a bit more. I've been able to re-create this on a physical device with my sample repo I posted above.

Device: Motorolla Droid Turbo 2
OS: Android 7.0

We also have reports of this happening on a different physical device (Samsung Note 8), but I do not have that device on hand to confirm. I can't speak to the Nexus 5, but can confirm that not all devices I've tested in the emulator exhibit the issue. I'll add some updates to this issue's description so it's clearer.

Hopefully this clarifies a bit, but I understand if you'd prefer to leave this issue closed -- it seems like a device-specific edge case. Thanks for reviewing and have a great day.
